docling/backend/msword_backend.py

```diff
@@ -273,6 +282,65 @@ def get_label_and_level(self, paragraph: Paragraph) -> tuple[str, Optional[int]]
 
         return label, None
 
+    @classmethod
+    def get_format_from_run(cls, run: Run) -> Formatting:
+        return Formatting(
+            bold=run.bold if run.bold is not None else False,
+            italic=run.italic if run.italic is not None else False,
+            underline=run.underline if run.underline is not None else False,
+        )
+
+    def format_paragraph(self, paragraph: Paragraph):
```

Comment:
Mark internal (i.e. not part of the public API):
```suggestion
    def _format_paragraph(self, paragraph: Paragraph):
```